### ISSUE TITLE
[HelpHub][6.4][headings]: Verse Block

### ISSUE THREAD

**Issue by zzap** [WordPress/Documentation-Issue-Tracker#1350]
Article:   https://wordpress.org/documentation/article/verse-block/

## Task

Make sure all headings are in sentence case (e.g. Block settings). 

## Directions
- Check all the headings in the article (link above).
- Note all the titles which are in title case (e.g. Block Settings).
- Post these titles in the comment on this issue.
- Request review by stating in the comment that the issue is ready for review.

## Resources

- Read about headings capitalisation - [Checking headings capitalization](https://make.wordpress.org/docs/handbook/get-involved/getting-started-at-a-contributor-day/good-first-issues/#checking-headings-capitalization)

## Automation
- If you want, you can assign yourself to the issue and change the label for review by posting specific keywords in a comment. Read about our repo's automatisation [here](https://make.wordpress.org/docs/handbook/github-repository-and-projects/documentation-issue-tracker/#label-issues).

Thank you for contributing, have a cookie :cookie:

**Comment by Joe4510** [comment]
`/assign`

**Comment by Joe4510** [comment]
Headings not in sentence case:

- Verse Block
- Block Toolbar
- Block Settings

**Comment by Joe4510** [comment]
Please `/review` @abhansnuk, @zzap

**Comment by nalininsbs** [comment]
@abhansnuk there are a couple more at the top. Wider than the capitalization, I have the same question about copyright on this especially as name of author is not fully seen.

**Comment by abhansnuk** [comment]
Yes I agree. I can't find information on whether there is permission for extracts to be used like this. However, as this a Random House publication and the author's name is not clearly seen in the screenshot, it is a good question. 

A question wider than headlines. @zzap and @estelaris where there are screenshots quoting from books and poems which may be subject to copyright, is it allowed to use it like this? In this screenshot, the author's name is partly hidden by toolbar. The poem is by Maya Angelou, who died in 2014, and this may be still within copyright and need permission from Random House for republication. If there is a link to information about using published books and poetry in screenshots, this would be useful may be to include for screenshot and video tasks when choosing text to use? This may already be covered somewhere so signposting to it would be very appreciated. Thanks.
